__tests__/app.test.js

```diff
@@ -0,0 +1,30 @@
+const db = require("../db/connection");
+const request = require("supertest");
+const seed = require("../db/seeds/seed");
+const data = require("../db/data/test-data/index");
+const app = require("../app");
+
+beforeEach(() => {
+  return seed(data);
+});
+
+afterAll(() => db.end());
+
+describe("GET /api/topics", () => {
+  it("200: should respond with a list of all topics and with each having a slug and description property", () => {
+    return request(app)
+      .get("/api/topics")
+      .expect(200)
+      .then(({ body }) => {
+        const { topics } = body;
+        expect(topics).toHaveLength(3);
+        topics.forEach((topic) => {
+          expect(topic).toHaveProperty("slug");
+          expect(topic).toHaveProperty("description");
+        });
+      });
+  });
+  it("400: should respond with a bad request message when passed a path that does not exist", () => {
```

Comment:
Your description says 400 but you're testing for 404, which one makes sense here? You also say you're testing for a message, but you're not - try to bring your description in line with you test 🧪